RELEASE-NOTES.md

```diff
@@ -1,5 +1,82 @@
 # Prefect Release Notes
-     
+
+## Release 2.10.13
+
+### Improvements to projects-based deployments
+
+![prefect deploy output with interactive cron schedule](https://github.com/PrefectHQ/prefect/assets/12350579/c94f45e6-3b7a-4356-84cd-f36a29f0415c)
+
+In this release we've made project-based deployments easier to use for first time users! You can now run `prefect deploy` without first initializing a project. If you run `prefect deploy` without a project initialized, the CLI will generate a default pull step that your worker can use to retrieve your flow code when executing scheduled flow runs. The `prefect deploy` command will also prompt you with schedule options and guide you through configuring the schedule making it even easier to schedule your flows!
```

Comment:
"The `prefect deploy` command will also prompt you with schedule options and guide you through configuring the schedule making it even easier to schedule your flows!"

I think the word "schedule" 3 times is too many here. 

How about

"The `prefect deploy` command will also prompt you with scheduling options, making it even easier to schedule your flows!"

Reply:
or
"The prefect deploy command will also prompt and guide you with scheduling options.."